app/code/Magento/Wishlist/view/frontend/web/js/add-to-wishlist.js

```diff
@@ -30,8 +31,10 @@ define([
         _bind: function () {
             var options = this.options,
                 dataUpdateFunc = '_updateWishlistData',
+                validateProductQty = '_validateWishlistQty',
                 changeCustomOption = 'change ' + options.customOptionsInfo,
                 changeQty = 'change ' + options.qtyInfo,
+                updateWishlist = 'click ' + options.actionElement,
```

Comment:
failed test https://public-results-storage-prod.magento-testing-service.engineering/reports/magento/magento2/pull/25641/0a4e4127615ba161e7e73f26a6e16db6/Statics/allure-report-ce/index.html#suites/5533c1ade71e7287a27c7a2c90cb7724/85783273d3bd5f09/

Error - 'updateWishlist' is assigned a value but never used. Allowed unused vars must match /config/. (no-unused-vars)

Cheers, 
Karla

Reply:
Hi! Thanks for help. This var is used in line 51, just like any other event.

Reply:
Yeah, seems like some issue with tests :)